bricksrc/equipment.py

```diff
@@ -439,6 +439,28 @@
     "AHU": {
         "tags": [TAG.Equipment, TAG.AHU],
         "subclasses": {
+            "Dedicated_Outdoor_Air_System": {
```

Comment:
What do you think about using `Dedicated_Outdoor_Air_System_Unit` so that it is clear from looking at the term that this is an Equipment and not a System?

Reply:
Yeah, I agree with your suggestion to make it clearer.